app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -13,6 +13,7 @@ sealed class RouteAction : Action {
     object ItemList : RouteAction()
     object Login : RouteAction()
     object SettingList : RouteAction()
+    object FxaAccount : RouteAction()
```

Comment:
Nit: naming. `FxA` stands for `Firefox Account`.

Should this be: `FxASettings`? or `FxALogin`?